apps/client/src/views/cuesheet/cuesheet-table/cuesheet-table-elements/EventRow.tsx

```diff
@@ -60,13 +60,9 @@ function EventRow(props: PropsWithChildren<EventRowProps>) {
     >
       {showActionMenu && (
         <td className={style.actionColumn}>
-          <MenuButton
-            as={IconButton}
-            size='sm'
-            aria-label='Options'
-            icon={<IoEllipsisHorizontal />}
-            variant='ontime-subtle'
-          />
+          <IconButton size='sm' aria-label='Options' variant='ontime-subtle'>
```

Comment:
the menu doesnt seem to be showing up for me

Reply:
Fixed with https://github.com/cpvalente/ontime/pull/1438/commits/8f05aa23bb742fc5b695ed2530c4eb5abd3bb84f